Modules/socketmodule.c

```diff
@@ -7071,16 +7071,24 @@ _socket_socket_if_nametoindex_impl(PySocketSockObject *self, PyObject *oname)
 static PyObject *
 socket_if_indextoname(PyObject *self, PyObject *arg)
 {
+    unsigned long index_long;
+    char name[IF_NAMESIZE + 1];
```

Comment:
I suggest to move this variable just before if_indextoname() call, and write `unsigned long index_long = ...;` if it fits into 80 columns.